runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansConfig.java

```diff
@@ -0,0 +1,87 @@
+package net.runelite.client.plugins.grotesqueguardians;
+
+import java.awt.Color;
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("grotesqueguardians")
+public interface GrotesqueGuardiansConfig extends Config
+{
+	@ConfigItem(
+		keyName = "displayTextOverlays",
+		name = "Display Text Overlays",
+		description = "Configures the display of text overlays",
+		position = 1
+	)
+	default boolean displayTextOverlays()
```

Comment:
Is this really necessary? the text is mostly not useful at all, just the tiles are important arent they, otherwise it will be just a lot of clutter

Reply:
Yes, seeing as if you don’t have the text, you will be unable to decipher which object is being spawned due to the fact that the tile indicators cover up the miniscule shadows in the beginning phase of it’s animations.

Reply:
just differentiate the colors more and remove shadows then.

Reply:
If need be, i will set defaults and remove the config. options if you desire so.